### PR TITLE
Fix statues' description not having a reference to their inability to regenerate. (#11448)

### PR DESCRIPTION
**Pull request by NormalPerson7**
This fixes https://crawl.develz.org/mantis/view.php?id=11448.

The first three commits can be ignored --  I still don't know how to clear irrelevant commits from a pull request. The only commit which does anything is the one labelled so.

### CODE REVIEW

File: crawl-ref/source/dat/des/portals/wizlab.des [hunk 1]
```diff
@@ -1895,4 +1895,4 @@ ccccccccccc.....cccccc....c*...*c....ccccc''...........''ccc
 ccccccccccccc.rcccccccccccccbPbcccccccccccc'''''''''''''cccc
 ccccccccccccccccccccccccccccbbbccccccccccccccccccccccccccccc
 cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
-ENDMAP
+ENDMAP
```
Comment (NormalPerson7) [inline crawl-ref/source/dat/des/portals/wizlab.des]: no idea what this is

File: crawl-ref/source/mon-data.h [hunk 2]
```diff
@@ -5909,7 +5909,7 @@ DUMMY(MONS_GOLEM, '8', LIGHTGREY, "golem", TILEP_MONS_IRON_GOLEM)
 
 { // always redefined
     MONS_STATUE, '8', LIGHTGREY, "statue",
-    M_STATIONARY | M_SPEAKS | M_ARCHER | M_DONT_MELEE | M_NO_POLY_TO,
+    M_STATIONARY | M_SPEAKS | M_ARCHER | M_DONT_MELEE | M_NO_POLY_TO | M_NO_REGEN,
```
Comment (aidanholm) [inline crawl-ref/source/mon-data.h]: `mons_can_regenerate()` checks the monster class's flags as a fallback, so you should be able to remove all other `M_NO_REGEN` flags and just keep the one on `MONS_STATUE`. Haven't tested this though.

Comment (NormalPerson7) [reply crawl-ref/source/mon-data.h]: I tested this and it does not appear to work - Only `MONS_STATUE` and its derivatives keep the "It cannot regenerate" text, all other statues do not get it.

**Comment by aidanholm** [comment]
Thanks for the fix, manually merged at https://github.com/crawl/crawl/commit/c608dd7fdd4d

**Pull request closed without merging**